src/HomeWork3.java

```diff
@@ -1,34 +1,37 @@
-class Letter {
-    public static void main(String[] args) {
-
-        String text = "Python";
-        char a = text.charAt(0);
-        System.out.println(a);
-        char b = text.charAt(text.length() - 1);
-        System.out.println(b);
-
+public class HomeWork3 {
+    public static int calculateAbsoluteDifference(int a, int b) {
+       return (-1) * (a - b);
```

Comment:
```
System.out.println(calculateAbsoluteDifference(5, 8)); // prints 3 correct
System.out.println(calculateAbsoluteDifference(8, 5)); // prints -3 which is incorrect, not an absolute value
System.out.println(calculateAbsoluteDifference(8, -8)); // prints -16 whis is also incorrect, not an absolute value
```
You solution works only when the first value is greater than the second value.
Tip: Look into what methods `Math` class has.